x-pack/test/functional/apps/ml/anomaly_detection/saved_search_job.ts

```diff
@@ -127,14 +127,14 @@ export default function ({ getService }: FtrProviderContext) {
           frontCardTitle: 'AAL',
         },
         row: {
-          recordCount: '34,415',
+          recordCount: '86,274',
           memoryStatus: 'ok',
           jobState: 'closed',
           datafeedState: 'stopped',
           latestTimestamp: '2016-02-11 23:59:54',
         },
         counts: {
-          processed_record_count: '34,415',
+          processed_record_count: '86,274',
```

Comment:
Not sure if this change was intended. But it's failing on my local deployment and on CI. I think we should revert this.

Reply:
Reverted here 958b660